src/main/java/edu/harvard/iq/dataverse/api/Admin.java

```diff
@@ -997,6 +997,26 @@ public Response fixMissingOriginalTypes() {
 
 		return ok(info);
 	}
+        
+    @Path("datafiles/integrity/fixmissingoriginalsizes")
```

Comment:
@landreev can you please document this new API endpoint in the guides?

Reply:
I have documented it in the Native API guide, with the other file-related API calls. Also added the release notes specific to the issue (doc/release-notes/5323-saved-original-size).

Reply:
(thanks :)